test/lib/jdk/test/lib/hprof/parser/Reader.java

```diff
@@ -95,8 +96,8 @@ public static Snapshot readFile(String heapFile, boolean callStack,
                 return r.read();
             } else if ((access = GzipRandomAccess.getAccess(heapFile, 16)) != null) {
                 in.close();
-                try (PositionDataInputStream in2 = new PositionDataInputStream(
-                        new BufferedInputStream(access.asStream(0)))) {
+                try (BufferedInputStream bis2 = new BufferedInputStream(access.asStream(0));
+                        PositionDataInputStream in2 = new PositionDataInputStream(bis2)) {
```

Comment:
Please fix indentation so that the type names align.

Also minor nit: the '2' suffix is not needed for any of the variables now as they are scoped to the try-block.

Reply:
Hi David,
     Removing the `2` suffix here makes build error, as they are conflict with the outer try() at line 88.  Thanks.